docs/openstack/stop_openstack_services.md

```diff
@@ -71,6 +79,13 @@ there are no ongoing  operations.
 3- Make sure all the services are stopped.
 4- Repeat steps 1-3 for compute hosts (workloads running on dataplane will not be affected)
 
+Additionally, remove the conflicting repositories (and packages, in case of a
```

Comment:
actually, this is too early to become a point of no return. I should move that step before starting EDPM adoption, which is an agreed point of no return instead.

Reply:
done in the roles split commit